src/lib/custom-animations/PressableScale.tsx

```diff
@@ -32,27 +31,25 @@ export function PressableScale({
   const scale = useSharedValue(1)
 
   const animatedStyle = useAnimatedStyle(() => ({
-    transform: [{scale: scale.value}],
+    transform: [{scale: scale.get()}],
   }))
 
   return (
     <AnimatedPressable
       accessibilityRole="button"
       onPressIn={e => {
-        'worklet'
         if (onPressIn) {
-          runOnJS(onPressIn)(e)
+          onPressIn(e)
         }
         cancelAnimation(scale)
-        scale.value = withTiming(targetScale, {duration: 100})
+        scale.set(() => withTiming(targetScale, {duration: 100}))
```

Comment:
is `cancelAnimation` blocking? do they need to be grouped together into a single `runOnUI` call?

Reply:
i think it is. don't want to change this right now, i need to better understand what `cancelAnimation` does. we've heard we're not supposed to use it like this but i do remember some need for it. let's not touch for now?